### PR TITLE
Meeting minutes 2024-11-07

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2024/2024-11-07-coordination.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    - Particularly for career paths and training
+    - This is very positive and goes beyond individual endorsement
+    - We have contacted other experiments to propose the same (waiting for feedback)
+
+Liz: will your experiment also make a submission from the software and computing area?
+
+- ATLAS are - Claire is meeting them tomorrow
+- LHCb are
+
+HSF's paper will be a *complimentary one* to the experiments.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2024/2024-11-07-coordination.md:44]:
```suggestion
HSF's paper will be a *complementary one* to the experiments.
```

File: organization/_posts/2024/2024-11-07-coordination.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+#### Activity Area Preparations
+
+##### Physics Generators
+
+Need to have a meeting between the conveners and the SG. Sapta will organise this. It's urgent to have the mini-workshop this year and there's not much time left.
+
+See also [Oliver's CHEP plenary](https://indico.cern.ch/event/1338689/contributions/6054977/), which was a great introduction. Negative weights and NNLO are urgent topics. Steve G can arrange a contribution from neutrino side.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2024/2024-11-07-coordination.md:52]:
```suggestion
See also [Oliver's CHEP plenary](https://indico.cern.ch/event/1338689/contributions/6054977/), which was a great introduction. Negative weights and NNLO are urgent topics. Steve G can arrange a contribution from the neutrino side.
```

**Review by eduardo-rodrigues — APPROVED** [review]
Thank you. (I made 2 trivial suggestions.)

**Comment by graeme-a-stewart** [comment]
Thanks for the checks and fixes, @eduardo-rodrigues

**Pull request merged**